cloudbackup_mon.sh

```diff
@@ -82,19 +96,21 @@ fi
 
 # Get username and agentid
 username=`cat /etc/driveclient/bootstrap.json | grep Username | awk '{print $3}' | sed -e 's/"//g' -e 's/,//g'`
-agentid=`cat /etc/driveclient/bootstrap.json | grep AgentId | awk '{print $3}' | sed -e 's/,//g'`
+agentid=`cat /etc/driveclient/bootstrap.json | grep AgentId\" | awk '{print $3}' | sed -e 's/,//g'`
```

Comment:
This is to remove the line containing: "AgentIdv2" from the json, which resulted in the 'agentid' bash variable having two lines in it.